srcpkgs/the-way/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'the-way'
+pkgname=the-way
+version=0.14.4
+revision=1
+archs="i686 x86_64"
```

Comment:
could you explain this?

Reply:
I think that was from the xnew template I used.

Maybe we should change xnew upstream to not include that line by default?

Reply:
> could you explain this?

I'm removing it.